pkg/virt-launcher/virtwrap/network/selinux_context_executor.go

```diff
@@ -89,12 +72,31 @@ func setVirtLauncherSELinuxContext(virtLauncherPID int) error {
 	return nil
 }
 
+func setVirtHandlerSELinuxSocketContext(virtLauncherPID int) error {
+	virtLauncherSELinuxLabel, err := getProcessCurrentSELinuxLabel(virtLauncherPID)
+	if err != nil {
+		return fmt.Errorf("error reading virt-launcher %d selinux label. Reason: %v", virtLauncherPID, err)
+	}
+
+	runtime.LockOSThread()
+	if err := selinux.SetSocketLabel(virtLauncherSELinuxLabel); err != nil {
+		return fmt.Errorf("failed to set selinux socket context to %s. Reason: %v", virtLauncherSELinuxLabel, err)
+	}
+	return nil
+}
+
 func resetVirtHandlerSELinuxContext() error {
 	err := selinux.SetExecLabel(virtHandlerSELinuxLabel)
 	runtime.UnlockOSThread()
 	return err
 }
 
+func resetVirtHandlerSELinuxSocketContext() error {
```

Comment:
I would call the method - `resetVirtHandlerSELinuxSocketLabel`. I'm not sure how the word "context" is related here.